### PR TITLE
update dockercompose setup

### PR DESCRIPTION
**Pull request by warren830**
# Summary

1. Add FAQ about how to use external db and grafana when using devlake by docker-compose
2. Remove anything related to pg

### CODE REVIEW

File: docs/GettingStarted/DockerComposeSetup.md [hunk 1]
```diff
@@ -33,8 +33,87 @@ sidebar_position: 1
...
 
-<br/>
+<br/> 
 
+## FAQ
+### 1. Can I use an external Database service instead of running Database in Docker?
+   
+    Yes, you can simply comment out mysql/grafana part in docker-compose.yml and update some configurations in .env befor you run `docker compose up -d`, here are the steps:
+
+1. Comment out mysql part:
```
Comment (Startrekzky) [inline docs/GettingStarted/DockerComposeSetup.md]: Can you remove the point number before the question? It's quite confusing now.
image

File: docs/GettingStarted/DockerComposeSetup.md [hunk 2]
```diff
@@ -33,8 +33,87 @@ sidebar_position: 1
...
+      MYSQL_ROOT_PASSWORD: admin
+      MYSQL_DATABASE: lake
+      MYSQL_USER: merico
+      MYSQL_PASSWORD: merico
+    command:
+      --character-set-server=utf8mb4
+      --collation-server=utf8mb4_bin
+```
+
+2. Comment out mysql volume:
```
Comment (Startrekzky) [inline docs/GettingStarted/DockerComposeSetup.md]: Text change:

Comment out the 'mysql' volume:

File: docs/GettingStarted/DockerComposeSetup.md [hunk 3]
```diff
@@ -33,8 +33,87 @@ sidebar_position: 1
 
 Support for database schema migration was introduced to DevLake in v0.10.0. From v0.10.0 onwards, users can upgrade their instance smoothly to a newer version. However, versions prior to v0.10.0 do not support upgrading to a newer version with a different database schema.
 
-<br/>
+<br/> 
 
+## FAQ
+### 1. Can I use an external Database service instead of running Database in Docker?
+   
+    Yes, you can simply comment out mysql/grafana part in docker-compose.yml and update some configurations in .env befor you run `docker compose up -d`, here are the steps:
```
Comment (Startrekzky) [inline docs/GettingStarted/DockerComposeSetup.md]: Text change:

Yes, you can simply comment out the 'mysql' part in 'docker-compose.yml' and update some configurations in '.env' before you run `docker compose up -d`, here are the steps:

File: docs/GettingStarted/DockerComposeSetup.md [hunk 4]
```diff
@@ -33,8 +33,87 @@ sidebar_position: 1
...
+```
+
+2. Comment out mysql volume:
+
+```yaml
+volumes:
+   mysql-storage:
+```
+
+3. Comment out depends_on mysql:
```
Comment (Startrekzky) [inline docs/GettingStarted/DockerComposeSetup.md]: Text change:

Comment out the 'depends_on mysql' part:

File: docs/GettingStarted/DockerComposeSetup.md [hunk 5]
```diff
@@ -33,8 +33,87 @@ sidebar_position: 1
...
+
+```yaml
+    depends_on:
+      - mysql
+```
+
+4. Set DB_URL to your own db url in .env
```
Comment (Startrekzky) [inline docs/GettingStarted/DockerComposeSetup.md]: How? Can you give an example?

File: docs/GettingStarted/DockerComposeSetup.md [hunk 6]
```diff
@@ -33,8 +33,87 @@ sidebar_position: 1
...
+
+5. Final step: `docker compose up -d`
+
+### 2. Can I use an external Grafana instead of running Grafana in Docker?
+
+    Yes, you can simply comment out mysql/grafana part in docker-compose.yml and update some configurations in .env befor you run `docker compose up -d`, here are the steps:
```
Comment (Startrekzky) [inline docs/GettingStarted/DockerComposeSetup.md]: Text change:

Yes, you can simply comment out the 'grafana' part in docker-compose.yml and update some configurations in '.env' before you run `docker compose up -d`, here are the steps:

**Review by Startrekzky — CHANGES_REQUESTED** [review]

**Review by Startrekzky — DISMISSED** [review]
LGTM

File: docs/GettingStarted/DockerComposeSetup.md (hunk 1 above)
Comment (warren830) [reply docs/GettingStarted/DockerComposeSetup.md]: fixed

File: docs/GettingStarted/DockerComposeSetup.md (hunk 2 above)
Comment (warren830) [reply docs/GettingStarted/DockerComposeSetup.md]: fixed

File: docs/GettingStarted/DockerComposeSetup.md (hunk 3 above)
Comment (warren830) [reply docs/GettingStarted/DockerComposeSetup.md]: fixed

File: docs/GettingStarted/DockerComposeSetup.md (hunk 4 above)
Comment (warren830) [reply docs/GettingStarted/DockerComposeSetup.md]: fixed

**Review by Startrekzky — CHANGES_REQUESTED** [review]
Can you update the v0.15 docs accordingly?

**Review by Startrekzky — APPROVED** [review]
LGTM

**Pull request merged**